### DISCUSSION TITLE
About https://github.com/bwapi/bwapi#legal-legal

### DISCUSSION THREAD

**Discussion by BoomerangAide** [bwapi/bwapi#892] [Q&A] [answered]
Hello,

In the readme, there is a part recommending to "purchase a legitimate copy" of Starcraft.
Considering that the game is officially and legally available for free on https://starcraft.com/en-us#features, I think either removing that part or adding that it is available for free at the aforementioned url, or something else (maybe regarding that statement and the existence of Remastered and compatibility issues), could be a good idea.

**Comment by dgant** [comment, marked as answer]
Blizzard gave permission to redistribute 1.16.1 free for AI purposes: http://www.cs.mun.ca/~dchurchill/startcraft/scbw_bwapi440.zip

There's also a version on their FTP server with a machine learning-specific license, whose URL I don't have handy.

**Reply by heinermann** [reply]
In addition this section has been long removed from the develop branch.

**Reply by BoomerangAide** [reply]
So it will be updated on the default (apparently "main") branch eventually?
Then I guess it's ok.
Thanks for answering.
